ui/src/components/main.css

```diff
@@ -87,10 +87,16 @@ button {
   padding-bottom: 5px;
 }
 
+/* This means we don't render it at all. */
 .hidden {
   display: none;
 }
 
+/* This means it is still there, just invisible. */
```

Comment:
Why do we bother rendering the self button at all? Isn't it fine for it to be hidden instead of invisible if no one would ever click it?

Reply:
Hidden means it actually doesn't get rendered which means there won't be anything to the left that keeps it at the same width away from the edge as the other rows. This was the easiest option, instead of making a custom empty inline div or something.